phovea_server/util.py

```diff
@@ -42,9 +42,38 @@ def to_json(obj, *args, **kwargs):
   kwargs['ensure_ascii'] = False
 
   # Pandas JSON module has been deprecated and removed. UJson cannot convert numpy arrays, so it cannot be used here. The JSON used here does not support the `double_precision` keyword.
+  if isinstance(obj, dict) or isinstance(obj, list):
+    obj = _handle_nan_values(obj)
   return json.dumps(obj, cls=JSONExtensibleEncoder, *args, **kwargs)
 
 
+def _handle_nan_values(obj_to_convert):
```

Comment:
let's add typings for this function now that we can use them and a little description that you used this function because a JSON encoder did not work